lib/scope.js

```diff
@@ -504,6 +512,9 @@ module.exports = {
     });
 
     if ( !disabled ) {
+      // da's header covers the top of the section containing the fields which means
+      // sometimes it blocks nodes from clicks. Scroll item to the bottom instead.
+      await handle.evaluate( elem => { elem.scrollIntoView(false); return elem.scrollTop });
```

Comment:
Do we need to `return elem.scrollTop` here? We don't seem to be using that value.

Reply:
Added to #171, remove unused dev code